run-tests.sh

```diff
@@ -3,10 +3,14 @@
 #
 # Copyright (C) 2020 CERN.
 # Copyright (C) 2020 Northwestern University.
+# Copyright (C) 2021 TU Wien.
 #
 # Invenio App RDM is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+# Usage:
+#   env DB=postgresql ./run-tests.sh [pytest options and args...]
```

Comment:
Why `env DB=postgresql` ? Works without... :thinking:

Reply:
ah yeah, that's just an example
the magic happens in line 28: `${DB:-postgresql}` will look for the environment variable `${DB}` and use it if it's defined - and if it's not defined, it'll use `"postgresql"` as fallback

Reply:
Right and postgresql is the default anyway. `Usage` sounded like a "must", but it's a "may".

Reply:
i'll update the comment accordingly